src/SDKs/Authorization/Authorization.Tests/Graph/GraphManagementClient.cs

```diff
@@ -248,11 +248,12 @@ private HttpRequestMessage CreateRequest(string uri, HttpMethod method, string b
 
             if (this.testEnvironment != null && this.testEnvironment.TokenInfo != null)
             {
-                // Not Supported in current code
-                // var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
-                // httpRequest.Headers.Authorization = new AuthenticationHeaderValue(
-                //                                this.testEnvironment.AccessTokenType,
-                //                                this.testEnvironment.AuthorizationContext.AccessToken);
+				// Not Supported in current code
+				// var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
+				httpRequest.Headers.Authorization = new AuthenticationHeaderValue("Bearer",
+					"Put the access token for graph here for the tests to work in record mode");
```

Comment:
Why do you need to do this?  The test framework acquires both ARM and Graph tokens